tests/storage_pool/storage_pool_test.go

```diff
@@ -64,6 +64,7 @@ var _ = Describe("{StoragePoolExpandDiskResize}", func() {
 
 		// px will put a new request in a queue, but in this case we can't calculate the expected size,
 		// so need to wain until the ongoing operation is completed
+		time.Sleep(time.Second * 60)
```

Comment:
is this sleep intended or leftover from testing?

Reply:
yes, it's intended, as pool resize can take a while